herbert_core/utilities/maths/noisify.m

```diff
@@ -51,7 +52,7 @@
             ymax = max(abs(y(:)))
         end
         % make noise dy and add to y for output; make error bar for noise    
-        dy=(fac*ymax)*randn(size(y));   % st. dev. of randn is sigma=1
+        dy=(fac*ymax)*randfunc(size(y));% default: randn(size(y)); % st. dev. of randn is sigma=1
```

Comment:
Nested comments. Also the `default:` here just repeats the logic of the code in lines 37-41

Reply:
RH comment is the original comment on the randn call. LH comment is commenting on the use of randfunc. It seems worth ramming home that randn is the "normal" use-case, not just a standard option if the user can't think of anything more specific. It is always good to restate things in different ways to get the point across.

Reply:
Fair enough.
I would suggest pulling that whole inline comment out to the preceding line then:
```
% make noise ....
% default: normal distribution, stddev = 1 (randn(size(y)))
dy=(fac*ymax)*randfunc(size(y));
```
That contains the same information but means I don't have to remember what randn does and not having to parse nested comments
```
code; % comment1 % comment2
```